forwards_paper.tex

```diff
@@ -243,11 +259,13 @@ \subsection*{Tree sequences}
 mutations. The HDF5 file encoding the nodes, edges, sites and mutations (as
 described above) for this simulation consumed 157MiB of storage space. Using
 the \msprime\ Python API, the time required to load this file into memory was
-around 1.5 seconds, and the time required to iterate over all 1 million tree
+around 1.5 seconds, and the time required to iterate over all 1 million trees
 was 2.7 seconds. In contrast, recording the topological information in Newick
 format would require around 20 TiB and storing the genotype information
-in VCF would require about 1 TiB. Working with either the Newick or VCF encoding
-of this dataset would be exceedingly cumbersome, likely requiring several
+in VCF would require about 1 TiB --
+a compression factor of 200,000.
```

Comment:
I added this note for those who didn't notice that inbetween MiB and TiB comes GiB, and might think it was only a factor of 200.